ui/static/ui/js/require_config.js

```diff
@@ -9,6 +9,10 @@ var REQUIRE_PATHS = {
   react_infinite: "react-infinite/dist/react-infinite",
   lodash: "lodash/lodash",
   select2: "select2/dist/js/select2.full",
+  uri: "uri.js/src/URI",
+  punycode: "uri.js/src/punycode",
+  IPv6: "uri.js/src/IPv6",
+  SecondLevelDomains: "uri.js/src/SecondLevelDomains",
```

Comment:
Yes, the other three are required by `URI.js` and `URI.js` is used to parse the URL query string